app/_kong_plugins/datakit/index.md

```diff
@@ -512,6 +518,14 @@ columns:
   - title: Attributes
     key: attributes
 rows:
+  - nodetype: "`branch`"
+    inputs: "user-defined"
+    outputs: "`then`, `else`"
```

Comment:
Since `then` and `else` are configuration attributes instead of I/O fields, should we list them here in `outputs`?